lib/rules/validate-jsdoc.js

```diff
@@ -122,7 +122,7 @@ module.exports.prototype = {
                 }
 
                 validators = _this._rulesForTags[scope];
-                if (!validators || !node.jsdoc) {
+                if (!validators || !node.jsdoc || !node.jsdoc.valid) {
```

Comment:
Didn't find a better way to split rules into files ;-(

But feels like it will be changed soon.